networking/network_observability/installing-operators.adoc

```diff
@@ -5,7 +5,7 @@ include::_attributes/common-attributes.adoc[]
 :context: network_observability
 
 toc::[]
-Installing Loki is a prerequisite for using the Network Observability Operator. It is recommended to install Loki using the Loki Operator; therefore, these steps are documented below prior to the Network Observability Operator installation.
+Installing Loki is a recommended prerequisite for using the Network Observability Operator. Therefore, these steps are documented below prior to the Network Observability Operator installation.
```

Comment:
So should we mention `network-observability-without-loki.adoc` page here to let user choose the proper approach ?

Reply:
Agree a link is good idea here.